tests/test_raft_paper.rs

```diff
@@ -815,7 +815,7 @@ fn test_vote_request() {
         r.step(m).expect("");
         r.read_messages();
 
-        for _ in 0..r.get_election_timeout() * 2 {
+        for _ in 1..r.get_election_timeout() * 2 {
```

Comment:
@xiang90 
we panic here https://github.com/pingcap/tikv/blob/fa1a765288cf73e4039e36dc2c8f832fdc34ac91/tests/test_raft_paper.rs#L825

we get msg count 4, means trigger two election.